src/main/java/Task.java

```diff
@@ -0,0 +1,19 @@
+abstract public class Task {
```

Comment:
I like the use of an abstract Task class that all other tasks inherit from, makes it easier in the case we want to add future tasks

Reply:
Perhaps this can be further abstracted by placing all relevant code in a 'task' package